aiida/manage/tests/pytest_fixtures.py

```diff
@@ -118,55 +115,75 @@ def test_1(aiida_localhost):
 
 
 @pytest.fixture(scope='function')
-def aiida_local_code_factory(aiida_localhost):  # pylint: disable=redefined-outer-name
+def aiida_local_code_factory(aiida_localhost):
     """Get an AiiDA code on localhost.
 
     Searches in the PATH for a given executable and creates an AiiDA code with provided entry point.
 
     Usage::
 
       def test_1(aiida_local_code_factory):
-          code = aiida_local_code_factory('pw.x', 'quantumespresso.pw')
+          code = aiida_local_code_factory('quantumespresso.pw', '/usr/bin/pw.x')
           # use code for testing ...
 
-    :return: A function get_code(executable, entry_point) that returns the Code node.
+    :return: A function get_code(entry_point, executable) that returns the `Code` node.
     :rtype: object
     """
 
-    def get_code(entry_point, executable, computer=aiida_localhost, prepend_text=None, append_text=None):
+    def get_code(
+        entry_point,
+        executable,
+        computer=aiida_localhost,
+        label=None,
+        prepend_text=None,
+        append_text=None,
+        check_executable=True
```

Comment:
My only comment is that we should be a bit careful not to let the parameters of this function explode.
Is it really necessary to use this fixture to set up a code pointing to a non-existing path?

Reply:
I would say yes, in most test case scenarios, the executable is never actually run so who cares if it can actually be found. So to be honest I am not sure why the check is there, but in order to not break backwards compatibility I put in a flag.

Reply:
I have reverted the change for now